app/controllers/application_controller.rb

```diff
@@ -222,6 +222,12 @@ def hide_banner
   def set_current_user
     User.current_user = logged_in_as_admin? ? current_admin : current_user
     @current_user = current_user
+    unless current_user.nil?
+      @current_user_subscriptions_count,@current_user_visible_work_count,@current_user_bookmarks_count,@current_user_owned_collections_count,@current_user_challenge_signups_count,@current_user_offer_assignments,@current_user_unposted_works_size=
+             Rails.cache.fetch("user_menu_counts_#{current_user.id}",
+                               expires_in: ArchiveConfig.DASHBOARD_MENU_CACHE_TIME,
+                               race_condition_ttl: 5) {"#{current_user.subscriptions.count},#{current_user.visible_work_count},#{current_user.bookmarks.count},#{current_user.owned_collections.count},#{current_user.challenge_signups.count},#{current_user.offer_assignments.undefaulted.count+current_user.pinch_hit_assignments.undefaulted.count},#{current_user.unposted_works.size}"}.split(",").map(&:to_i)
```

Comment:
Surrounding space missing for operator '+'.<br>Space missing inside {.<br>Space missing inside }.